backend/onyx/background/celery/tasks/indexing/tasks.py

```diff
@@ -162,9 +166,14 @@ def get_unfenced_index_attempt_ids(db_session: Session, r: redis.Redis) -> list[
     bind=True,
 )
 def check_for_indexing(self: Task, *, tenant_id: str | None) -> int | None:
+    """a lightweight task used to kick off indexing tasks.
+    Occcasionally does some validation of existing state to clear up error conditions"""
+    time_start = time.monotonic()
+
     tasks_created = 0
     locked = False
     r = get_redis_client(tenant_id=tenant_id)
```

Comment:
typically prefer r to be more verbose like redis_client = get_redis_client(tenant_id=tenant_id)